platform/util/platform-util-uuidgenerator/util-uuidgenerator-impl/pom.xml

```diff
@@ -45,7 +45,6 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Transitive>true</Embed-Transitive>
```

Comment:
❓ Odd change to include as part of this PR. Was it on purpose?

Reply:
Yup. It should not have been set and setting it actually ends up embedding more than we need especially since we now add default test dependencies.